ephyviewer/base.py

```diff
@@ -191,6 +194,15 @@ def viewer(self):
     def source(self):
         return self._viewer().source
 
+    def apply_xsize_zoom(self, xmove):
```

Comment:
I am not sur if this is the good place for this.
Because some viewer inherits this but have nothing to do with this drag and zoom.

But for now it is OK.

Reply:
I think you're right. I did it that way because originally `apply_xsize_zoom` was a method of `TraceViewer_ParamController`, and I was trying to generalize it by moving it from there to a parent class common to all param controllers. In this case, the direct parent of `TraceViewer_ParamController` is `Base_MultiChannel_ParamController`. However, I ended up putting `apply_xsize_zoom` into its grandparent (`Base_ParamController`) instead because one of the viewers in which I wanted this to work (`EpochEncoder`) has a param controller that inherits from `Base_ParamController` directly, rather than from `Base_MultiChannel_ParamController`.

So, yes, I agree it isn't the best place for it. There's probably a better way. I'll add that to my list.